docker-compose.yml

```diff
@@ -117,14 +139,20 @@ services:
 
 volumes:
   app_data:
+  repos:
+  authorized_keys:
   postgres:
   redis:
   bundle:
   node_modules:
   packs:
   packs_test:
-  log:
   tmp:
+  log:
+    driver_opts:
+      type: none
+      device: ${PWD}/log
```

Comment:
I just did a pull to try this PR out again, and ran into an issue here. I think I ran into this before but forgot to mention it, my bad.

Got a warning `The PWD variable is not set. Defaulting to a blank string.` I don't think this is the intended behaviour? A quick Google search yielded https://stackoverflow.com/questions/41948232/docker-compose-wont-find-pwd-environment-variable.

Reply:
Unfortunately using a `.` for the current directory here won't work because we're defining a directory path on the host machine (not the containers) and so a full path is required. The issue is that we're relying on an environment variable here that is not set for windows. Or at least not for `cmd` (https://stackoverflow.com/a/41489151/5992438). 

This volume doesn't really need to be a bind mount, I just put it there so that it would be easier to see the logs without having to go into the containers. But really that's not a huge deal and we're doing that in most other cases anyway.

I think the best solution here is to remove `log` as a bind mount and just have it as a regular volume.

Reply:
Okay cool, thanks for the info. Yeah why don't you change this to a regular volume.